src/DMS/Service/Meetup/Response/SelfParsingResponse.php

```diff
@@ -44,29 +30,21 @@ public function getData()
     }
 
     /**
-     * @param array $metadata
-     */
-    public function setMetadata($metadata)
-    {
-        $this->metadata = $metadata;
-    }
-
-    /**
-     * @return array
+     * Makes Meetup API Data available on the Data attribute
      */
-    public function getMetadata()
```

Comment:
The `SelfParsingResponse` don't use the metadata at all so I just moved to the class that actually uses it (`MultiResultResponse`).

I find it hard to consider this as a BC break since its just `null` on instances of the `SelfParsingResponse`.